.github/workflows/latest-release.yml

```diff
@@ -1,31 +1,70 @@
+name: Test KubeArmor image before pushing
 on:
   push:
     # Sequence of patterns matched against refs/tags
     branches:
       - 'main'
 
-name: Latest version release
+env: 
+  TEST_TAG: kubearmor/kubearmor:test
 
 jobs:
   build:
     name: Create KubeArmor Release - 18.04
-    if: github.repository == 'kubearmor/KubeArmor'
+    # if: github.repository == 'kubearmor/KubeArmor'
```

Comment:
is this because you need to test in your repo?

Reply:
Yes, but still I think we can remove this command as repository will always be the same and it will also help others to test the action in their fork without requiring any changes.

Reply:
then, let's remove it.